components/brave_ads/core/test/data/catalog_with_multiple_campaigns.json

```diff
@@ -185,18 +178,17 @@
                 },
                 "wallpapers": [
                   {
+                    "conditionMatchers": [
+                      {
+                        "prefPath": "brave.today.opted_in",
+                        "condition": "1"
+                      }
+                    ],
                     "focalPoint": {
                       "x": 1000,
                       "y": 1200
                     },
                     "imageUrl": "https://brave.com/2/test2.jpg"
-                  },
-                  {
```

Comment:
In theory, this is valid if it is non-rewards to have more than one wallpaper, correct?

Reply:
Ah yes these are already removed, I just need to push the change.